utils/docker/Dockerfile.code_scanning

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright 2018-2020, Intel Corporation
+#
+# 'recipe' for Docker for code scanning based on RPM building.
+#
+
+# Pull base image
+FROM fedora:latest
+MAINTAINER daos-stack <daos@daos.groups.io>
+
+# use same UID as host and default value of 1000 if not specified
+ARG UID=1000
+
+# Install basic tools
+RUN dnf -y install mock\ \<\ 2 mock-core-configs\ \<\ 32 make \
+                   rpm-build curl createrepo rpmlint redhat-lsb-core git \
+                   python-srpm-macros rpmdevtools
```

Comment:
Does this really need any of this?

Reply:
No, I was leaving them there to recycle the common stuff from the mockbuild Dockerfile.

Reply:
But what common stuff?  If none of those packages are needed, then this Dockerfile is just `fedora:latest` plus `bandit` isn't it?

Reply:
I suppose we could have a _python+git_ common layer here upon which both the _mockbuild_ and this Dockerfile build, but it's probably a pretty low value optimisation.